### PR TITLE
Reword automated decision-making

### LINKED ISSUE #123: Clarify "automated decision-making" in privacy policy
https://github.com/whatwg/sg/pull/120, concerning our website whatwg.org, uses the phrasing "It does not use other tracking technologies and it does not use automated decision-making."

For a layperson, this not easy to interpret, as pointed out by @domenic in https://github.com/whatwg/sg/pull/120#discussion_r416843033.

I speculated that it's in reference to GDPR, but have since learned that the wording is intended to not *only* be about GDPR, but also other privacy regulation.

We should clarify this wording, and @othermaciej had some ideas here.

### PR DESCRIPTION
**Pull request by annevk**
Closes #123.

### CODE REVIEW

**Review by domenic — APPROVED** [review]
Much clearer!

**Review by foolip — APPROVED** [review]
I like it!

**Review by travisleithead — APPROVED** [review]
I'm OK with the change as-is.

**Review by othermaciej — APPROVED** [review]
Some of our legal representatives specifically wanted to include the phrase "automated decision-making" because it reflects language from GDRP (I guess the idea is that the privacy policy implies compliance with GDRP if it does this?). However, the new language seems much more clear and a synonym for what is intended by GDPR. So this seems fine as-is, or we could find a way to have the new language and the original phrase somehow (perhaps using parens).

File: Privacy Policy.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ We have appropriate technical and organizational security measures in place to p
 
 ## Cookies and other technologies
 
-Some parts of our website allow logging in. In such cases, our website uses cookies to retain your logged-in state. It does not use other tracking technologies and it does not use automated decision-making.
+Some parts of our website allow logging in. In such cases, our website uses cookies to retain your logged-in state. It does not use other tracking technologies and it does not use algorithms or profiling to make decisions that significantly affect you.
```
Comment (annevk) [inline Privacy Policy.md:41]:
```suggestion
Some parts of our website allow logging in. In such cases, our website uses cookies to retain your logged-in state. It does not use other tracking technologies and it does not use algorithms or profiling to make decisions that significantly affect you (also known as automated decision-making).
```

**Review by travisleithead — APPROVED** [review]
New phrasing LGTM.

**Pull request merged**